tests/test_quixstreams/fixtures.py

```diff
@@ -72,19 +79,12 @@ def factory(
         auto_commit_enable: bool = True,
         extra_config: dict = None,
```

Comment:
BTW, this seems wrong
```suggestion
        extra_config: Optional[dict] = None,
```